test/e2e/storage/testsuites/provisioning.go

```diff
@@ -337,7 +337,10 @@ func SetupStorageClass(
 			framework.ExpectNoError(err)
 			clearComputedStorageClass = func() {
 				framework.Logf("deleting storage class %s", computedStorageClass.Name)
-				framework.ExpectNoError(client.StorageV1().StorageClasses().Delete(context.TODO(), computedStorageClass.Name, metav1.DeleteOptions{}))
+				err := client.StorageV1().StorageClasses().Delete(context.TODO(), computedStorageClass.Name, metav1.DeleteOptions{})
+				if err != nil && !apierrors.IsNotFound(err) {
+					framework.ExpectNoError(err, "delete storage class")
+				}
```

Comment:
This change was necessary because the new capacity test calls the function twice, once as part of the test and then through `defer`. I think ignoring "NotFound" errors is a common pattern when deleting.

Reply:
I understand this is making the cleanup idempotent, good idea